packages/grid/_modules_/grid/hooks/features/export/seralizers/csvSeraliser.ts

```diff
@@ -31,13 +31,16 @@ export function serialiseRow(
   return mappedRow;
 }
 
-export function buildCSV(
-  columns: GridColumns,
-  rows: Map<GridRowId, GridRowModel>,
-  selectedRows: Record<string, GridRowId>,
-  getCellValue: (id: GridRowId, field: string) => GridCellValue,
-  delimiterCharacter: CsvDelimiterCharacter = ',',
-): string {
+type BuildCSVOptions = {
```

Comment:
It should be an interface

Reply:
@dtassone Ah, right, my bad. Could you push this as a new commit directly on the PR so we can move forward with it?